backend/resilience/views.py

```
@@ -0,0 +1,108 @@
+from os import name
```

Ruff (0.8.2)

`os.name` imported but unused

Remove unused import: `os.name`

(F401)